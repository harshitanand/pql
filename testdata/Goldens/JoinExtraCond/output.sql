WITH "__subquery0" AS (SELECT * FROM "Tokens"),
     "__subquery1" AS (SELECT * FROM (SELECT DISTINCT * FROM "LexResults") AS "$left" JOIN "__subquery0" AS "$right" ON ("$left"."Kind" = "$right"."Kind") AND (coalesce("Value" <> 'bar', FALSE)) ORDER BY "SpanStart" ASC NULLS FIRST)
SELECT "TokenConstant" AS "TokenConstant", "Value" AS "Value" FROM "__subquery1";
