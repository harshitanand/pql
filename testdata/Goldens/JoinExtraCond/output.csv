TokenConstant,Value
TokenIdentifier,foo
TokenPipe,
